app/controllers/items_controller.rb

```diff
@@ -18,6 +18,14 @@ def create
   end
 
   def update
+    @item = Item.find(params[:id])
+    @item.update_attributes(item_params)
```

Comment:
`update_attributes` will save to the database, which is probably not your intention since you're saving on line 24.  You can either use this on line 24 instead of `save` OR use [assign_attributes](http://devdocs.io/rails/activerecord/attributeassignment#method-i-assign_attributes) which is probably what you intended for this line to do.